backend-models/service-tests/sectionedservice-test.txt

```diff
@@ -0,0 +1,551 @@
+sectionedservice.php
+
+----------------------------------
+getGroupsTest() line 100:
+----------------------------------
+
+*********
+ Pre-req
+*********
+
+(checklogin) IS TRUE
+
+//// PRE-REQ Values /////
```

Comment:
I'm a bit confused are, the sent values meant to be part of the pre-req or is there no pre-req?

Reply:
I presume the values are for the choose course that will be displayed